src/rhumb.js

```diff
@@ -60,34 +56,38 @@ function create (){
       , more = !!parts.length
       , peek
 
-    
-    if(isArr(part)){
+    if (Array.isArray(part)) {
+      if (node.leaf) {
+        throw new Error('Ambiguous')
```

Comment:
Should probably be `throw new Error("Ambiguity")` to be consistent. Bad, but consistent, at least.